UnityProject/Assets/Scripts/Managers/MatrixManager/MatrixManager.cs

```diff
@@ -340,6 +340,13 @@ private static bool IsInMatrix(Vector3Int worldPos, bool isServer, MatrixInfo ma
 			WorldTo = Worldorigin + (Vector3) (direction.normalized * distance);
 		}
 
+		if ((WorldTo - Worldorigin).Value.magnitude > 25)
+		{
+			Logger.LogError(" Limit exceeded on raycast, Look at stack trace for What caused it "); //Meant to catch up stuff that's been naughty and doing stuff like 900 tile Ray casts
+			return new CustomPhysicsHit();
+		}
```

Comment:
Isnt the distance parameter also used to control the range? Also 25 seems maybe too small

Reply:
25 is about off screen and little bit more, on the edges up and below it plenty extra

Reply:
What about the distance parameter?

Reply:
Distance parameter is just used to , automatically convert it into a Line  cast from a Raycast

Reply:
Could probably allow it to go through but just log the magnitude?

Reply:
but lag tho?